docs/src/user-guide/configuring/language-options.md

```diff
@@ -180,21 +182,19 @@ Globals can be disabled with the string `"off"`. For example, in an environment
 }
 ```
 
-For historical reasons, the boolean value `false` and the string value `"readable"` are equivalent to `"readonly"`. Similarly, the boolean value `true` and the string value `"writeable"` are equivalent to `"writable"`. However, the use of older values is deprecated.
+For historical reasons, the boolean value `false` and the string value `"readable"` are equivalent to `"readonly"`. Similarly, the boolean value `true` and the string value `"writeable"` are equivalent to `"writable"`. However, the use of these older values is deprecated.
```

Comment:
```suggestion
For historical reasons, the boolean value `false` and the string value `"readable"` are equivalent to `"readonly"`. Similarly, the boolean value `true` and the string value `"writeable"` are equivalent to `"writable"`. However, the use of these older values are deprecated.
```

Reply:
'is' is correct because verb agrees with 'the use'